src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -300,12 +300,16 @@ ov::Any CompiledModel::get_property(const std::string& name) const {
             config.fcDynamicQuantizationGroupSize);
     } else if (name == ov::hint::kv_cache_precision) {
         return decltype(ov::hint::kv_cache_precision)::value_type(config.kvCachePrecision);
+    } else if (name == ov::cache_encryption) {
+        return decltype(ov::cache_encryption)::value_type(config.cacheEncrypt);
```

Comment:
It seems cacheEncrypt/cache_decryption doesn't need read operation?

Reply:
updated.